### PR TITLE
[Keymap] Add qmk_configurator keymap chavdai40

### PR DESCRIPTION
**Pull request by dvorak55**
Hi, I want to add keymap/c/chavdai40_default.json.

Firmware of chavdai40 PR is qmk/qmk_firmware#9820(merged)

### CODE REVIEW

File: public/keymaps/c/chavdai40_default.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "keyboard": "chavdai40",
+    "keymap": "default",
+    "layout": "LAYOUT_42key",
```
Comment (noroadsleft) [inline public/keymaps/c/chavdai40_default.json:4]: Please change the layout macro to `LAYOUT_44key`, and add source commit hash for version tracking.

```suggestion
    "commit": "245797f6b998fb4f52df61e38151c80b05615740",
    "layout": "LAYOUT_44key",
```

File: public/keymaps/c/chavdai40_default.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "layers": [
+      [
+        "KC_TAB",  "KC_Q",  "KC_W",  "KC_E",  "KC_R",  "KC_T",  "KC_Y",   "KC_U",  "KC_I",     "KC_O",    "KC_P",     "KC_MINS", "KC_BSPC",
+        "KC_LCTL", "KC_A",  "KC_S",  "KC_D",  "KC_F",  "KC_G",  "KC_H",   "KC_J",  "KC_K",     "KC_L",    "KC_SCLN",  "KC_ENT",
+        "KC_LSFT", "KC_Z",  "KC_X",  "KC_C",  "KC_V",  "KC_B",  "KC_N",   "KC_M",  "KC_COMM",  "KC_DOT",  "KC_SLSH",  "MO(0)",
+        "KC_LALT", "KC_LGUI",                                   "KC_SPC",                      "KC_RGUI", "KC_RALT"
```
Comment (noroadsleft) [inline public/keymaps/c/chavdai40_default.json:10]:
```suggestion
        "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_DEL",
        "KC_LALT", "KC_LGUI",                                  "MO(1)",   "KC_SPC",                        "KC_RGUI", "KC_RALT", "KC_ESC"
```

File: public/keymaps/c/chavdai40_default.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      ],
+      [
+        "KC_GRV",  "KC_EXLM", "KC_AT",  "KC_HASH",  "KC_QUOT",  "KC_PERC",  "KC_CIRC",  "KC_AMPR",  "KC_ASTR",  "KC_LPRN",  "KC_RPRN",  "KC_LBRC",  "KC_RBRC",
+        "KC_TRNS", "KC_1",    "KC_2",   "KC_3",     "KC_4",     "KC_5",     "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",     "KC_TRNS",
+        "KC_TRNS", "KC_F2",   "KC_F3",  "KC_F4",    "KC_F5",    "KC_F6",    "KC_F7",    "KC_F8",    "KC_F9",    "KC_F10",   "KC_EQL",   "KC_TRNS",
+        "KC_TRNS", "KC_TRNS",                                               "KC_TRNS",                          "KC_TRNS",  "KC_TRNS"
```
Comment (noroadsleft) [inline public/keymaps/c/chavdai40_default.json:16]:
```suggestion
        "KC_TRNS", "KC_TRNS",                                  "KC_TRNS", "KC_TRNS",                       "KC_TRNS", "KC_TRNS", "KC_TRNS"
```

**Comment by dvorak55** [comment]
Thanks for noroadsleft!
I just committed suggestions of you.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**